src/item.cpp

```diff
@@ -8223,20 +8227,21 @@ int item::fill_with( const itype &contained, const int amount )
         contained_item.charges = 1;
     }
 
-    item_pocket *pocket = best_pocket( contained_item );
-    if( pocket == nullptr ) {
+    item_location loc;
+    std::pair<item_location, item_pocket *> pocket = best_pocket( contained_item, loc );
```

Comment:
```suggestion
    item_pocket *pocket = best_pocket( contained_item, loc ).second;
```
you _could_ do it this way to reduce a little bit of churn, though i am unsure on the value of doing that.